agent/util-scripts/pbench-register-tool-set

```diff
@@ -101,6 +101,9 @@ case "$toolset" in
 			nerrs=$nerrs+1
 		fi
 	done
+        ansible-galaxy install chaitanyaenr.ara
```

Comment:
I agree with @atheurer, can you explain what "ara" is more, and why it needs to live in register tool?